packages/host/src/api/api.ts

```diff
@@ -5,7 +5,7 @@ export const getHostQueries = (sdk: Sdk) => ({
   get: {
     settings: async () => {
       const result = await sdk.serverSettings();
-      return result.serverSettings;
+      return result?.serverSettings;
```

Comment:
so that I could get the original error if there was one, rather than throwing an error here due to `result` being undefined. Not actually used now, but still a good check to have in.